files/nrpe/check_openstack.py

```diff
@@ -931,21 +937,50 @@ class OSCinderAvailability(cinder.Client):
   options = dict()
 
   def __init__(self, options):
-    self.cinder = cinderclient.client.Client('2', session=keystone_session_v3(options))
+    self.cinder = cinderclient.client.Client('3', session=keystone_session_v3(options))
```

Comment:
Out of curiosity, was there a technical requirement for this change or is this more about just keeping up with the latest APIs?

Reply:
Asking because I'm not sure I follow why this is needed (it might still be good to change it, but then for consistency we could consider changing OSVolumeErrorCheck() API version too)

Reply:
When I updated the code, I tried to make it compatible with Python 3 and use latest API version. This prevents incompatibility with the new updates. I have tested and it works. However, we can return it to the original version.

Reply:
I agree with you about OSVolumeErrorCheck() API. I did not want to do big changes to make the task of reviewer easy.

Reply:
Cinder v2 API is deprecated since Queens so The Right Thing (TM) would be to just bump everything to v3.

I'm fine with either option:

1) Revert OSCinderAvailability() back to v2 since we don't need this change if introducing a standalone OSCinderServiceAvailability() method.

2) Bump all Cinder checks to use the v3 API.

Reply:
Done. Now all cinder APIs are version 3. Nova and Neutron still use previous version. If we plan to bump them, we need to check further.

Reply:
Cool! And everything ok in monitoring after provisioning to devel?

Reply:
Yes. I did that last time.